### ISSUE TITLE
[FEAT] Make save location configurable

### ISSUE THREAD

**Issue by denics** [NeonGeckoCom/skill-camera#47]
### Objective

It should be possible to define where to save the images.

### Initial Implementation Requirements

- configure the skill under ``.config/neon/skills/skill-camera``

**Comment by NeonDaniel** [comment]
I agree; probably just need a property in the skill that defaults to the current value. This property should also validate that the configured path is valid and writable by the user
